### PR TITLE
Add GDS questionnaire

### PR DESCRIPTION
**Pull request by NicolasRuth**

### CODE REVIEW

File: R/GDS.R [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+#' @param short_version (Scalar boolean) For the short version of the
+#' questionnaire set this to TRUE. Defaults to FALSE.
+#' Overridden by the \code{configuration_filepath} and \code{"subscales"} arguments.
+#' @param ... Further arguments to be passed to \code{\link{GDS}()}.
+#' @export
+GDS <- function(label = "GDS",
+                dict = psyquest::psyquest_dict,
+                subscales = c(),
+                short_version = FALSE,
+                configuration_filepath = NULL,
```
Comment (fmhoeger) [inline R/GDS.R]: Gibt es dafür auch einen GDS-Konfigurator wie bei GMS?

**Review by fmhoeger — CHANGES_REQUESTED** [review]

File: tests/testthat/apps/GDS_fullscale_DE/GDS_fullscale_DE.csv [hunk 2]
```diff
@@ -0,0 +1,27 @@
+"MID","Item","Response","Negative"
```
Comment (fmhoeger) [inline tests/testthat/apps/GDS_fullscale_DE/GDS_fullscale_DE.csv]: This file can also be deleted?!

**Review by fmhoeger — CHANGES_REQUESTED** [review]

File: tests/testthat/apps/GDS_fullscale_DE/GDS_fullscale_DE.csv (hunk 2 above)
Comment (NicolasRuth) [reply tests/testthat/apps/GDS_fullscale_DE/GDS_fullscale_DE.csv]: Yes, done. Test still works

File: man/GDS.Rd [hunk 3]
```diff
@@ -0,0 +1,45 @@
+% Generated by roxygen2: do not edit by hand
```
Comment (fmhoeger) [inline man/GDS.Rd:1]: You need to regenerate the Rd doc files by running devtools::check() in RStudio, as they still contain references to `short_version` and `configuration_filepath`.

**Review by fmhoeger — CHANGES_REQUESTED** [review]

File: man/GDS.Rd (hunk 3 above)
Comment (NicolasRuth) [reply man/GDS.Rd:1]: ok, done.

File: tests/testthat/apps/GDS_fullscale_DE/inst/extdata/GDS_items_to_subscales.csv [hunk 4]
```diff
@@ -0,0 +1,27 @@
+"item_id","subscale_id"
```
Comment (fmhoeger) [inline tests/testthat/apps/GDS_fullscale_DE/inst/extdata/GDS_items_to_subscales.csv]: Do we need this one?

**Review by fmhoeger — CHANGES_REQUESTED** [review]

File: tests/testthat/apps/GDS_fullscale_DE/inst/extdata/GDS_items_to_subscales.csv (hunk 4 above)
Comment (NicolasRuth) [reply tests/testthat/apps/GDS_fullscale_DE/inst/extdata/GDS_items_to_subscales.csv]: Don't think so. Deleted it.

**Pull request merged**